Header (#2)

renemorales opened the pull request:
- Reset default browser styles
- Add global styles
- Add router with routes / and /search
- Add global fonts
- Change app title and meta description
- Add globalStyle & theme

renemorales committed:
Header (#2)

- Reset default browser styles
- Add global styles
- Add router with routes / and /search
- Add global fonts
- Change app title and meta description
- Add globalStyle & theme

mentor-tara commented:
Great job :rocket: I'm a bit busy at the moment but I'll leave a review soon.

In the meantime, consider starting with the next task already to practice the professional Git workflow. [If you need a refresher, feel free to follow this section in the course again.](https://profy.dev/project/reddit-timer/while-waiting)[](ckma5leps3kko0b87rxtjjp24)

mentor-tara reviewed (changes requested):
Before I approve this PR I'd like to share a few tips with you.

**Please go through the list below and review your own code first.** Imagine you're the reviewer or a person who didn't see the code before. Ask yourself if everything is clearly written and understandable. Are components and variables properly named?

Reviewing your own code with this mindset is an important practice that will help you learn a lot and write high-quality code.

To make it a bit easier for you here are the tips I promised. Some have a link :link: attached where you can find more information. I'd recommend reading the **bold tips** in any case. Feel free to check the boxes when you think your code fits.

**General:**
- [ ] Use human readable URLs [:link:](https://profy.dev/project/reddit-timer/review-tips/human-readable-links)
- [ ] Use absolute instead of relative links: `<Link to="/search" />` [:link:](https://profy.dev/project/reddit-timer/review-tips/use-absolute-links)
- [ ] No components that solely copy props to their children [:link:](https://profy.dev/project/reddit-timer/review-tips/unnecessary-components)
- [ ] Optimize svg files with SVGOMG [:link:](https://profy.dev/project/reddit-timer/review-tips/optimize-svg-with-svgomg)
- [ ] Use react-router Link instead of HTML anchor tag [:link:](https://profy.dev/project/reddit-timer/review-tips/link-instead-of-anchor-tag)

**Styles:**
- [ ] **Where to put your styles** [:link:](https://profy.dev/project/reddit-timer/review-tips/where-to-put-styles)
- [ ] **Refrain from styling children with CSS selectors** [:link:](https://profy.dev/project/reddit-timer/review-tips/css-selectors)
- [ ] **Extract shared CSS values** [:link:](https://profy.dev/project/reddit-timer/review-tips/common-css-values-in-theme)
- [ ] Be careful with pixel-perfectness when using relative font sizes like `0.9em` [:link:](https://profy.dev/project/reddit-timer/review-tips/careful-with-relative-font-size)

**Tests:**
- [ ] **Use React Testing Library for testing** [:link:](https://profy.dev/project/reddit-timer/review-tips/use-react-testing-library)
- [ ] Don't write snapshot tests [:link:](https://profy.dev/project/reddit-timer/review-tips/no-snapshot-tests)
- [ ] **Prefer integration tests over unit tests** [:link:](https://profy.dev/project/reddit-timer/review-tips/no-unit-tests)
- [ ] Get the logo by role with `getByRole` [:link:](https://profy.dev/project/reddit-timer/review-tips/get-logo-by-role)
- [ ] How to test hash links [:link:](https://profy.dev/project/reddit-timer/review-tips/testing-hash-links)

Once you're done with reviewing your own code feel free to check the [example implementation](https://profy.dev/project/reddit-timer/code-examples/header-ae7e45f0-51b2-433a-b293-3bef0ae80cce). **But remember: It's an important exercise to review your code first on your own.**

When you think this PR is ready to be merged just let me know by requesting another review. I'll approve right away.[](ckmaavvyw3m7p0b358cxn7dzq)

Commit:
- Add header component
-  Add logo and links
- Review the code
- Fix merge conflicts
- Change the URL to search link

Commit:
Add new route

mentor-tara reviewed (approved):
Great work! :tada::tada::tada: You can merge the PR now.[](ckmabg2tk3mkv0b35j06077kr)
